Make.inc

```diff
@@ -898,12 +899,21 @@ LIBBLAS := -L$(build_shlibdir) -lopenblas
 LIBBLASNAME := libopenblas
 endif
 
-# OpenBLAS builds LAPACK as part of its build.
+# OpenBLAS builds cblas/LAPACK as part of its build.
 # We only need to build LAPACK if we are not using OpenBLAS.
 ifeq ($(USE_SYSTEM_BLAS), 0)
+LIBCBLAS := $(LIBBLAS)
+LIBCBLASNAME := $(LIBBLASNAME)
 LIBLAPACK := $(LIBBLAS)
 LIBLAPACKNAME := $(LIBBLASNAME)
 else
+ifeq ($(USE_SYSTEM_CBLAS), 1)
+LIBCBLAS ?= -lcblas
+LIBCBLASNAME ?= libcblas
+else
+LIBCBLAS := -L$(build_shlibdir) -lcblas $(LIBBLAS)
+LIBCBLASNAME := libcblas
```

Comment:
I think this may explain the CI failure: you set `LIBCBLASNAME` to something different from `LIBBLASNAME`, and below you check that they are different to decide whether to build `LIBCBLASNAME`. I think you should just remove the `else`, since without `USE_SYSTEM_CBLAS=1` Julia won't build a separate CBLAS.

Reply:
The point of my suggestion earlier is that perhaps we should always be linking against CBLAS rather than just BLAS, instead of as a special case since we are using the CBLAS ABI.

Reply:
@StefanKarpinski Do you think we really need CBLAS? See my previous comment: https://github.com/JuliaLang/julia/pull/29540#pullrequestreview-162252287

Reply:
Aren't we already using the OpenBLAS's CBLAS API when making BLAS calls?

Reply:
Dunno, but this PR only changes two calls.

Reply:
> two calls

Four

Reply:
Yes. We use the CBLAS version of the complex dot functions in https://github.com/JuliaLang/julia/blob/master/stdlib/LinearAlgebra/src/blas.jl#L293-L294 because the complex return value in the Fortran API didn't work. See https://github.com/JuliaLang/julia/pull/5291 and https://github.com/JuliaLang/julia/pull/5283#issuecomment-31493948. We use the Fortran API for everything else.

Reply:
OK. Might be worth retrying since it's been almost 5 years.